python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -253,12 +254,14 @@ def train_epochs(self, data_creator, epochs=1, batch_size=32, profile=False,
             val_loader = None
             val_steps = None
 
-        if callbacks is not None:
-            for callback in callbacks:
-                callback.set_model(self.given_models)
-                if hasattr(callback, "set_trainer"):
-                    callback.set_trainer(self)
-                callback.on_train_begin()
+        # Check uniqueness of the MainCallback
+        make_only_mainCallback(callbacks)
```

Comment:
What about check it in the estimator? The error message thrown by the main process may be easier to find?